android/app/src/test/java/com/zulipmobile/notifications/FcmMessageTest.kt

```diff
@@ -52,6 +52,19 @@ class FcmMessageTest : FcmMessageTestBase() {
     }
 }
 
+class RecipientTest : FcmMessageTestBase() {
+    @Test
+    fun `GroupPm#getPmUsersString gives the correct string`() {
+        expect.that(Recipient.GroupPm(pmUsers = setOf(123, 234, 345)).getPmUsersString())
+            .isEqualTo("123, 234, 345")
+    }
+    @Test
+    fun `GroupPm#getPmUsersString correctly reorders user ids`() {
+        expect.that(Recipient.GroupPm(pmUsers = setOf(234, 123, 345)).getPmUsersString())
+            .isEqualTo("123, 234, 345")
+    }
```

Comment:
This test can be made a bit stronger by testing that e.g. 123 sorts after 23 -- i.e. the sorting is of the numbers, and not a lexicographic sort of their ASCII representations.